versioned_docs/version-8.5/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -332,31 +332,33 @@ The actual configuration properties can be reviewed [in the Tasklist configurati
 
 5. For Connectors:
 
-The Connectors component will only accept a `jks`(Java KeyStore) certificate.
-If you have followed our previous recommendation of generating a TLS certificate using the OpenShift annotation, then you will have a `PKCS #1` certificate which the Connectors component will not accept. For convenience, here are a number of commands that will convert the `PKCS #1` certirficate, generated by OpenShift to `jks` format, that Connectors component accepts:
+The Connectors component will only accept a `jks` (Java KeyStore) certificate.
+If you have followed our previous recommendation of generating a TLS certificate using the OpenShift annotation, you will have a `PKCS #1` certificate the Connectors component will not accept.
+
+For convenience, below are a number of commands that will convert the `PKCS #1` certificate, generated by OpenShift to a `jks` format the Connectors component accepts:
 
 ```bash
-# Grab OpenShift generated TLS certificate
+#Grab OpenShift generated TLS certificate.
 kubectl get secret -n camunda camunda-zeebe-gateway -o jsonpath="{.data['tls\.crt']}" | base64 --decode > tls.crt
-# Grab OpenShift generated TLS key.
+#Grab OpenShift generated TLS key.
```

Comment:
@christinaausley Could you please let me know why you removed the space after the hash in the comments? It is a bash convention to have a space after the hash. I'm just asking out of curiosity 😄

Reply:
Cool 👍 I'll add this to the WCoE.